### PR TITLE
Adding modal dialogs to fault report buttons of items to make them more intuitive

### PR DESCRIPTION
**Pull request by viktorcsimma**
The first weeks have shown that many users did not understand how these buttons worked (e.g. they reported a fault for the wrong machine). This will hopefully help.

grafik

grafik

grafik

### CODE REVIEW

**Comment by BertalanD** [comment]
I like the initiative, hopefully we'll get fewer unintended triggers.

Another idea we could try is to show a modal dialog where users are required to write a short description of the problem (likely would also help Erika)

**Comment by viktorcsimma** [comment]
That’s a good idea:) I hope I’ll have the time to implement that at the weekend.

**Comment by viktorcsimma** [comment]
~@BertalanD and what if we simply redirected them to the [Faults](https://uran.eotvos.elte.hu/faults) page? The only issue is that admins would also need notifications about faults.~

~Though maybe we should not promote that page, as we are supposed to switch to the common university fault reporting system.~

File: resources/sass/materialize.scss [hunk 1]
```diff
@@ -301,6 +305,11 @@ body.dark a.btn {
     box-sizing: border-box;
 }
 
+// for footers of modal dialogs
+.modal .modal-footer {
+    height: unset;
+}
```
[the remark this answers is not in the document]
Comment (viktorcsimma) [reply resources/sass/materialize.scss]: Nice; thank you:)

**Comment by viktorcsimma** [comment]
Now, I've managed to implement the pop-up:) can you check it out again?

**Comment by BertalanD** [comment]
This still looks a bit weird in Safari:

image

**Comment by BertalanD** [comment]
Ah, solved. Just needed to recompile the assets.

**Review by BertalanD — APPROVED** [review]

**Pull request merged**